packet/context_processors.py

```diff
@@ -49,6 +51,18 @@ def get_rit_name(username):
         return username
 
 
+@lru_cache(maxsize=128)
+def get_rit_image(username):
```

Comment:
Maybe it'd make more sense to call this `get_gravatar()` instead? It's not pulling the image from any RIT service so it's not really an "RIT image".